src/Utils/Embed.php

```diff
@@ -38,29 +57,35 @@ function (ItemInterface $item) use ($url) {
                     $embed = (new BaseEmbed())->get($url);
                     $oembed = $embed->getOEmbed();
                 } catch (\Exception $e) {
+                    $this->logger->info('Embed:fetch: fetch failed: '.$e->getMessage());
                     $c = clone $this;
-                    unset($c->cache);
-                    unset($c->settings);
 
                     return $c;
                 }
 
-                $this->url = $url;
-                $this->title = $embed->title;
-                $this->description = $embed->description;
-                $this->image = (string) $embed->image;
-                $this->html = $this->cleanIframe($oembed->html('html'));
+                $c = clone $this;
+
+                $c->url = $url;
+                $c->title = $embed->title;
+                $c->description = $embed->description;
+                $c->image = (string) $embed->image;
+                $c->html = $this->cleanIframe($oembed->html('html'));
 
                 try {
-                    if (!$this->html && $embed->code) {
-                        $this->html = $this->cleanIframe($embed->code->html);
+                    if (!$c->html && $embed->code) {
+                        $c->html = $this->cleanIframe($embed->code->html);
                     }
                 } catch (\TypeError $e) {
+                    $this->logger->info('Embed:fetch: html prepare failed: '.$e->getMessage());
```

Comment:
I tried to set this logging to `debug` across Mbin, to avoid too much log pollution.. Hence the default log value for the dev mode is currently `info`.

Reply:
it is an error though, I would actually have expected this to be error rather than debug, but not familiar with the log generation

Reply:
should this one be downgraded to `debug` level then? 

the way I see it is that this a rather minor error, not a showstopper, but it's still an error and so it should be logged with a bit higher level compared to the standard debug/trace/diagnostic logs, and for this I think `info` is fine

Reply:
I agree with info, @nobodyatroot not sure if you've been running this and also asdfzdfj do either of your happen to know how often it comes up? If it's too much, we can always tone it down. I guess if an instance is having issues it might make it a bit spammy

Reply:
I have it run on my instance since yesterday and I haven't seen it even once. maybe @nobodyatroot might have more info as my single user selfhosted instance and is a rather quiet one

actually, given the current code I don't event know what could trigger this exception in the first place

Reply:
oops, I misread this comment as ln 60, that's the info one I might expect to see it fetching fails

Reply:
i think this one happens so infrequently (but super annoying when it does and we notice more when it f-s up and puts a NSFW image in there) that it's going to need extended live testing. i'd be fine with merging it and let it run wild for awhile as it doesn't seem to hurt anything right now, can't say i notice any increased log file size... only 6 MB in the last 24 hours.

Reply:
Sure ok. Let's leave it like this.